FileAPI/blob/Blob-constructor.any.js

```diff
@@ -290,7 +290,9 @@ test_blob(function() {
     new Int16Array([0x4150, 0x5353]),
     new Uint32Array([0x53534150]),
     new Int32Array([0x53534150]),
-    new Float16Array([2.65625, 58.59375]),
+    typeof Float16Array === 'undefined'
```

Comment:
Do we have any precedent for "TODO(#nnn)" comments so we remember to go back and update these?

Reply:
Yeah, we should add a comment. I don't think we have a consistent pattern however.

But given that Float64Array is split out below, perhaps we can do that for Float16Array as well? That would solve it entirely. The only follow-up then would potentially be merging it all into a single test again at some point in the future.

Reply:
Split it into its own test.

Reply:
>The only follow-up then would potentially be merging it all into a single test again at some point in the future.

Or completely split them into all separate tests, it's easier to notice from wpt.fyi that way.